### ISSUE TITLE
More details about the MT results of mBART

### ISSUE THREAD

**Issue by luofuli** [pytorch/fairseq#2625]
#### What is your question?
Hello, I want to know more details about the MT experimental settings and results.
1. Can you provide the train updates and batch size when fine-tuning on WMT16 ro-en (bitext-only) and WMT16 eo-en (+back translation) in Table 4?
2.  In Table 3,  which test set of WMT en-de do you use? And also the corresponding train updates and batch size?

I think those details are important for me to reproduce the MT results of mBART.  Thx~

@SunbowLiu

**Comment by alphadl** [comment]
Looking forward to more details too @SunbowLiu
